examples/gno.land/r/demo/boards2/flag.gno

```diff
@@ -0,0 +1,47 @@
+package boards2
+
+import (
+	"std"
+	"strconv"
+)
+
+var flagThreshold = 3
```

Comment:
This should be 1 by default.

To have in mind for a following issue, we should support to configure the threshold by board, for example when a board is created though an option/param. For this issue we should add support to change each board flag threshold though a public realm function. Different boards could have different moderation requirements regarding flagging. Having this in mind we should have a flag threshold value in the Board type so posts can read it from it's parent board.

The global default `flagThreshold` should be a const:

```suggestion
const flagThreshold = 1
```

Reply:
@jeronimoalbi 

> For this issue we should add support to change each board flag threshold though a public realm function. Different boards could have different moderation requirements regarding flagging.

Probably updating flag threshold should be a separate permission and shouldn't be done by user. Should this be done now or can be done as a further improvement later (this is mentioned as future improvement in the issue)?

Reply:
We will do it later 👍 

I mentioned it just for context and because it might be relevant to know while you are implementing the flagging.

Reply:
The important part is that the threshold should be assigned to each board, if it makes sense